### PR TITLE
Use Elasticsearch 6

### PR DESCRIPTION
**Pull request by lfdebrux**
We are fully migrated to Elasticsearch 6 on production, we should migrate our local systems as well.

### CODE REVIEW

File: config/docker-compose.yml [hunk 1]
```diff
@@ -1,19 +1,26 @@
 version: '3.5'
 
+volumes:
+  postgres-data:
+  elasticsearch-data:
```
Comment (katstevens) [inline config/docker-compose.yml]: What is dmrunner doing with its data currently?

Comment (lfdebrux) [reply config/docker-compose.yml]: Currently it's stored in the container, so when you change the image you lose the data

Comment (katstevens) [reply config/docker-compose.yml]: So if we actually did want to wipe the data, how would we go about it? Maybe worth sticking something in the README?

Comment (lfdebrux) [reply config/docker-compose.yml]: I added something to the commit so the `make data` command will wipe as before

**Pull request merged**